app/src/main/java/com/lla/panicbutton/ui/fragments/RecordingsFragment.kt

```diff
@@ -31,11 +33,23 @@ class RecordingsFragment : Fragment(R.layout.fragment_recordings)  {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
         setupRecyclerView()
+
+        uploadButton.setOnClickListener{
+            findNavController().navigate(R.id.action_recordingsFragment_to_setupFragment)
```

Comment:
since it's no longer the "setup fragment" really, can we change it everywhere to be "uploadFragment"? I think it'll add a lot of changes to this PR, so maybe in another one.

Reply:
we can create an issue to remember

Reply:
https://github.com/LLA-panic/panic-button/issues/11

Reply:
I can change that right now actually because I think android studio refactors it everywhere. If not, I'll just change it manually.